src/VAP/LineDetector/DetectionLine.cs

```diff
@@ -103,6 +150,12 @@ public double getFractionInForeground(Bitmap mask)
         }
 
 
+        /// <summary>
+        /// Finds the box with the maximum overlap fraction with this <see cref="DetectionLine"/>.
+        /// </summary>
+        /// <param name="boxes">The list of <see cref="Box"/> objects to check, representing the bounding boxes of items in frame.</param>
+        /// <param name="mask">A mask detailing the precise layout of items in the frame using black to indicate vacant space, and white to indicate occupied space.</param>
+        /// <returns>Returns a <see cref="Tuple{double, Box}"/> containing both the maximum overlap fraction found, and the <see cref="Box"/> associated with that overlap.</returns>
```

Comment:
> `<see cref="Tuple{double, Box}"/>`

This isn't a supported form in documentation comments today. A request to add support for this was filed in https://github.com/dotnet/csharplang/issues/401, but has not been implemented so far.

Reply:
I'll probably change that to <c> then, since it's an external reference.